README.md

```diff
@@ -1,6 +1,7 @@
 # GoSub: Gateway to Optimized Searching and Unlimited Browsing
 
-This repository holds the GoSub HTML5 parser/tokenizer. It is a standalone library that can be used by other projects but will ultimately be used by the GoSub browser. See the [About](#about) section for more information.
+This repository holds the GoSub HTML5 parser/tokenizer. It is a standalone library that can be used by other projects

```

Comment:
Seems like my editor wraps with newlines.

Reply:
GitHub doesn't render markdown properly when newlines are introduced.  For Markdown (only), you want lines to wrap.

Reply:
I'll update the pr to wrap them.